server/accounts.go

```diff
@@ -704,6 +714,9 @@ type ServiceLatency struct {
 	TotalLatency   time.Duration `json:"total"`
 }
 
+// ServiceLatencyType is the NATS Event Type for ServiceLatency
+const ServiceLatencyType = "io.nats.server.metric.v1.service_latency"
```

Comment:
Possibly dumb question, but for type. These are never coming from client apps and only from the NATS system itself. Should we remove server?

io.nats.metrics.v1.service_latency

Reply:
I anticipate we'll have these elsewhere, like we have `io.nats.jetstream` now, I dont want to not have it and regret it later basically

Reply:
ok, can I offer system then? Server just does not feel right to me.

`io.nat.system.metrics.v1.service_latency`

I do think should be metrics though no?